guides/common/modules/hardware_storage_prerequisites.adoc

```diff
@@ -36,20 +36,20 @@ The runtime size was measured with Red Hat Enterprise Linux 6, 7, and 8 reposito
 
 == Storage Guidelines
 
-Consider the following guidelines when installing Satellite Server to increase efficiency.
+Consider the following guidelines when installing {ProjectServer} to increase efficiency.
 
 
-* Because most Satellite and Capsule Server data is stored within the `/var` directory, mounting `/var` on LVM storage can help the system to scale.
+* Because most {Project} and {SmartProxyServer} data is stored within the `/var` directory, mounting `/var` on LVM storage can help the system to scale.
 
-* For the `/var/lib/pulp/` and `/var/lib/mongodb/` directories, use high-bandwidth, low-latency storage, and solid state drives (SSD) rather than hard disk drives (HDD). As Red{nbsp}Hat Satellite has many operations that are I/O intensive, using high latency, low-bandwidth storage causes performance degradation. Ensure your installation has a speed in the range 60 - 80 Megabytes per second. You can use the `fio` tool to get this data. See the Red{nbsp}Hat Knowledgebase solution https://access.redhat.com/solutions/3397771[Impact of Disk Speed on Satellite 6 Operations] for more information on using the `fio` tool.
+* For the `/var/lib/pulp/` and `/var/lib/mongodb/` directories, use high-bandwidth, low-latency storage, and solid state drives (SSD) rather than hard disk drives (HDD). As {ProjectName} has many operations that are I/O intensive, using high latency, low-bandwidth storage causes performance degradation. Ensure your installation has a speed in the range 60 - 80 Megabytes per second. You can use the `fio` tool to get this data. See the Red{nbsp}Hat Knowledgebase solution https://access.redhat.com/solutions/3397771[Impact of Disk Speed on {ProjectX} Operations] for more information on using the `fio` tool.
```

Comment:
I guess any KBase links would be inside an ifeval for downstream only?

Reply:
outside the scope of this PR, but we are figuring out what to do about them. This is an issue that has come up before and I don't quite know what to do yet.